testsuite/MDAnalysisTests/analysis/test_hbonds.py

```diff
@@ -88,6 +92,18 @@ def test_generate_table(self):
         assert_array_equal(h.table.acceptor_resnm, self.values['acceptor_resnm'])
 
     @staticmethod
+    def test_atoms_too_far(self):
+        pdb = '''TITLE     Two atoms far away
+CRYST1   52.763   52.763   52.763  90.00  90.00  90.00 P 1           1
+MODEL         1
+ATOM      1  N   LEU     1      32.310  13.778  14.372  1.00  0.00      SYST N 0
+ATOM      2  OW  SOL     2       3.024   4.456   4.147  1.00  0.00      SYST H 0'''
+
+        u = MDAnalysis.Universe(BytesIO(pdb.encode()), format="pdb")
+        h = MDAnalysis.analysis.hbonds.HydrogenBondAnalysis(u, 'resname SOL', 'protein')
+        h.run(verbose=False)
+        assert_equal(h.timeseries, [[]])
+
```

Comment:
There has to be a @staticmethod in front of the test_true_traj() function; Travis fails https://travis-ci.org/MDAnalysis/mdanalysis/jobs/227170225 , too, and QC flags it as a [critical issue](https://www.quantifiedcode.com/app/project/gh:MDAnalysis:mdanalysis/diff/401c36636a5c12d250d2edd08ef44f678fe8ba45/2804d48f5bcc914ed461f64ba80e96c8ffb9a135/issues?groups=code_patterns%3A3bECxdfc).